notebooks/feature_engineering.ipynb

```diff
@@ -0,0 +1,2790 @@
+{
```

Comment:
Seems like most features you built are in this notebook, make transformer classes to extract them.